x-pack/qa/password-protected-keystore/src/test/resources/ssl/ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I'm just curious: we used to generate these types of materials within build.gradle in other qa tests: why are we now committing them instead of generating on the fly?

Reply:
We used to depend on BouncyCastle to generate keys and certificates as JDK doesn't expose the necessary APIs for the certificate generation (https://bugs.openjdk.java.net/browse/JDK-8058778) and we removed BouncyCastle dependency as part of the FIPS 140 effort